src/flypad-backend/src/FlyPadBackend.cpp

```diff
@@ -5,6 +5,7 @@
 #include "Aircraft/AircraftPreset.h"
 #include "Lighting/LightPreset.h"
 #include "Pushback/Pushback.h"
+#include "ThirdParty/ThirdParty.h"
```

Comment:
Why did you call it third party - and Selcal? Is it generic enough to do other things? Do you plan to extend this?

Reply:
No, if it is expandable then Selcal is not a good name. But how about OnlineAtc or AtcServices or something like this. Says more than ThirdParty

Reply:
I think there is a quiproquo :D

You are talking about Selcal feature then ThirdParty. 

Selcal is the real name of the feature therefore I cannot change it. Or are you talking about the PR's name ?

Regarding the class's name, yes I can find something more understandable

Reply:
Renamed into ATCServices